MekHQ/src/mekhq/gui/enums/PersonnelTableModelColumn.java

```diff
@@ -668,6 +674,34 @@ public String getCellValue(final Campaign campaign, final PersonnelMarket person
                 // Final fallback return of nothing
                 return "-";
             }
+            case SHIP_TRANSPORT:
+                if (person.getUnit() !=null){
+                    if (person.getUnit().getTransportShipAssignment() !=null ) {
+                        return person.getUnit().getTransportShipAssignment().getTransportShip().getName();
+                    }
+                }
+                if (!person.getTechUnits().isEmpty()) {
+                        Unit unit = person.getTechUnits().get(0);
+                        if (unit.getTransportShipAssignment() !=null ) {
+                            return unit.getTransportShipAssignment().getTransportShip().getName();
+                        }else return "-";
+                }
+                return "-";
+
+            case TACTICAL_TRANSPORT:
+                if (person.getUnit() !=null) {
+                    if (person.getUnit().getTacticalTransportAssignment() != null) {
+                        return person.getUnit().getTacticalTransportAssignment().getTransport().getName();
+                    }
+                }
+                if (!person.getTechUnits().isEmpty()) {
+                    Unit unit = person.getTechUnits().get(0);
+                    if (unit.getTacticalTransportAssignment() !=null ) {
```

Comment:
TechUnits are the units which this person is a *tech* of, like if they are a MekTech and they work in 5 mechs all those meks would show up as `techUnits` for this person. So I think that saying that it would be incorrect to assign a transport to a tech using the techUnits assignment.